### PR TITLE
Rename to Oasis ROSE Wallet, except in sidebar

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

File: .changelog/1879.breaking.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Rename to ROSE Wallet
```
Comment (buberdds) [inline .changelog/1879.breaking.md:1]: - Rename to Oasis ROSE Wallet
- why it's breaking? we do not plan to bump major version with this update, right?

Comment (lukaw3d) [reply .changelog/1879.breaking.md:1]: Im not sure. Process then?

**Comment by buberdds** [comment]
We should take into account meta tags like title or og:title and use full product name there.

**Comment by lukaw3d** [comment]
> We should take into account meta tags like title or og:title and use full product name there.

aw, i liked the tab name:
image

currently google would look like this:
image

currently slack would look like this:
image

@tjanez should we change these to Oasis ROSE Wallet?

**Review by buberdds — APPROVED** [review]
from code perspective it looks good. Product wise I would wait for Tadej.

File: .changelog/1879.breaking.md (hunk 1 above)
Comment (tjanez) [reply .changelog/1879.breaking.md:1]: I would classify it as a feature and not breaking.

We will bump major version when we release extension and web from the same codebase.

**Comment by tjanez** [comment]
> @tjanez should we change these to Oasis ROSE Wallet?

Yes, change these to Oasis ROSE Wallet.

**Pull request merged**